zuul.d/projects.yaml

```diff
@@ -208,6 +208,11 @@
     templates:
       - publish-to-galaxy-3pci
 
+- project:
```

Comment:
We might need to add ``default-branch: main`` here ? TBD once we change the default branch from master to main.

Reply:
Default branch is `main` in the repo now